src/interpreter_funcs/voxelize.rs

```diff
@@ -158,13 +160,22 @@ impl Func for FuncVoxelize {
         log: &mut dyn FnMut(LogMessage),
     ) -> Result<Value, FuncError> {
         let mesh = args[0].unwrap_mesh();
-        let voxel_dimensions = args[1].unwrap_float3();
+        let voxel_dimensions = Vector3::from(args[1].unwrap_float3());
         let growth_iterations = args[2].unwrap_uint();
         let fill = args[3].unwrap_boolean();
         let error_if_large = args[4].unwrap_boolean();
         let analyze_bbox = args[5].unwrap_boolean();
         let analyze_mesh = args[6].unwrap_boolean();
 
+        if voxel_dimensions
+            .iter()
+            .any(|dimension| approx::relative_eq!(*dimension, 0.0))
+        {
+            let error = FuncError::new(FuncVoxelizeError::VoxelDimensionsZero);
+            log(LogMessage::error(format!("Error: {}", error)));
+            return Err(error);
+        }
+
```

Comment:
Check if the vector dimensions are above zero. Should never happen thanks to the GUI settings. Maybe can be turned into an assert and panic if it happens. I didn't want to risk it at the exhibition though.

Reply:
That's a relief! Better safe than sorry!